README.md

```diff
@@ -55,35 +50,43 @@ cmake ..  -DCMAKE_INSTALL_PREFIX=/usr  -DCMAKE_BUILD_TYPE=Debug && make -j4
 
 For Arch based distributions an [AUR] package is available.
 
-## Notes
+## Notes and Issues
+
+* With LXQt 2.0 no different location for labwc configuration is used anymore as
+all components have wayland support now.
+
+* Bottom and right panels have some alignment issues for tooltips and and menu popups.
 
-* In "Session Settings" **uncheck** the "Panel" module and check only
-  "Panel X11/Wayland" in order to use different settings for `lxqt-panel`
-  on wayland. Panels with a "Desktop switch" widget will crash - do not add it (fixed in LXQt 1.4).
-* **Taskbar**: The only taskbar which displays window titles with eliding is `swfbar`; waybar can show well only app names and/or icons; yatbfw displays icons only.
 * LXQt lock settings do not work yet. For screenlock settings under wayland
-  please check `~/.config/lxqt-wayland/labwc/autostart`.
-* **Update**: To hide the panel with fullscreen applications using `<action name="ToggleAlwaysOnBottom"/>` in `~/.config/lxqt-wayland/labwc/rc.xml` "Windowrules" section is recommended
+  please check `~/.config/labwc/autostart`.
+  
 * Some X11-only applications (example: redshift) in autostart
   can lead to high CPU usage under wayland.
-* All options for labwc with LXQt are handled in `~/.config/lxqt-wayland/labwc/`.
+
 * LXQt Mouse/touchpad and keyboard settings do not work but keyboard layout(s)
   will be imported or set at first run.See
-  `~/.config/lxqt-wayland/labwc/environment` and the "libinput" section in `rc.xml`.
-* Mouse cursor and size are synced and can be set using "Appearance" settings, session restart required. GTK settings have to be updated after changes.
-* Global shortcuts are handled exclusively in `~/.config/lxqt-wayland/labwc/rc.xml`.
-* Refer also to this file for some lxqt-settings (e.g. positions)
-* Some panel plugins do not work on wayland, see [lxqt-panel] for more information.
+  `~/.config/labwc/environment` and the "libinput" section in `rc.xml`.
+  
+* Mouse cursor and size are synced and can be set using "Appearance" settings,
+session restart required. GTK settings have to be updated after changes.
```

Comment:
On latest master the session restart should not be required anymore, on 0.7.1 it is required.

Reply:
Thanks, I forgot. But `labwc -r` is needed, no?

Reply:
> Thanks, I forgot. But `labwc -r` is needed, no?

yes

Reply:
Done, thanks.